dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -96,6 +96,13 @@
     E2207( "Data start date must be before data end date" ),
     E2208( "Non-numeric data values encountered during outlier value detection" ),
 
+    /* Followup value analysis */
+    E2300( "At least one data element or data set must be specified" ),
```

Comment:
I am wondering if it is better to reuse the error codes from outlier detection where possible, and just change the comment/header to "data quality" or similar. I know there is a small risk of conflicting needs later, but I think it could be better overall to reuse error codes where feasible to avoid getting a ton of them/equal ones.

Reply:
Reused outlier detection where possible.